core/services/beacon/settings.py

```diff
@@ -125,3 +125,28 @@ def get_interface_or_create_default(self, name: str) -> Interface:
         )
         self.interfaces.append(new)
         return new
+
+
+class SettingsV2(SettingsV1):
+    VERSION = 2
+
+    def __init__(self, *args: str, **kwargs: int) -> None:
+        super().__init__(*args, **kwargs)
+        self.VERSION = 2
+
+    def migrate(self, data: Dict[str, Any]) -> None:
+        if data["VERSION"] == SettingsV2.VERSION:
+            return
+
+        if data["VERSION"] < SettingsV2.VERSION:
+            super().migrate(data)
+
+        data["default"]["domain_names"] = [domain for domain in data["default"]["domain_names"] if domain != "blueos"]
```

Comment:
I believe that we should stay with `blueos.local` for cable ethernet connections only, since this is a reliable connection source for `blueos.local`, different of wireless that may have problems.